tests/common/database_fixture.cpp

```diff
@@ -114,8 +114,23 @@ database_fixture::database_fixture()
       options.insert(std::make_pair("track-account", boost::program_options::variable_value(track_account, false)));
    }
 
-   ahplugin->plugin_set_app(&app);
-   ahplugin->plugin_initialize(options);
+   if(boost::unit_test::framework::current_test_case().p_name.value == "es1") {
```

Comment:
in here it will be good to filter by the test suite but i was not able to find how to do it. all elasticsearch tests will need those so we will need to add all the names, will be better if we can filter by the entire suite. any help on this will be appreciated.

Reply:
there are some member functions available to get the test suite but they are not available in boost 1.57. i think is not a big deal to it this way by now until something better and compatible is found.